udata/migrations/2023-02-08-rename-internal-dates.py

```diff
@@ -0,0 +1,46 @@
+'''
+The purpose here is to change the dates name in the dataset and resource model.
+'''
+import logging
+
+from mongoengine.connection import get_db
+from udata.models import Dataset
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
```

Comment:
It seems to work nicely! :muscle:
Did you try renaming harvest field (on dataset and resource) to `last_modified` for total coherence? :p